### ISSUE TITLE
Git LFS Issues

### ISSUE THREAD

**Issue by KeijiBranshi** [USCCACS/GEARS#5]
It seems we keep hitting our data caps due to our extremely large .DATA files. We should figure out an alternative way to host these (if at all). 

```
Downloading UnityGEARS/Editor/Assets/Demo2-VirtualConfocalMicroscopy/GeometryData.zip (110 MB)
Error downloading object: UnityGEARS/Editor/Assets/Demo2-VirtualConfocalMicroscopy/GeometryData.zip (030fecd): Smudge error: Error downloading UnityGEARS/Editor/Assets/Demo2-VirtualConfocalMicroscopy/GeometryData.zip (030fecd7a6d460fc0db4cebcb1838f2602b63d7e1de76a6b1090997e35605716): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
```

EDIT: added the actual error message I get on a `git clone`

**Comment by MekWarrior** [comment]
If it is truly needed, we could store it in a cloud bucket and then just have a script download and verify against a hash (if file integrity was a concern).

**Comment by KeijiBranshi** [comment]
Yeah, that seems like the most straight forward choice to me. @phwuser @KenichiNomura does CACS have an existing cloud storage choice we could make use of?

**Comment by phwuser** [comment]
Not sure. Please ask Ken-ichi, Brandon.

**Comment by KeijiBranshi** [comment]
Fixed in #7
